### ISSUE TITLE
Feature request: add `did_resolveDid` endpoint

### ISSUE THREAD

**Issue by insipx** [xmtp/xps-gateway#69]
### Is your feature request related to a problem?

We should include this endpoint in XPS since it already exists in DID resolver. We can merge the RPC methods as in the `rpc namespaces` example [here](https://github.com/xmtp/didethresolver/tree/main/examples/rpc_namespaces)

**Comment by jac18281828** [comment]
Does it make sense to just deploy didethresolver from the infrastructure repo rather than add this here?

**Comment by insipx** [comment]
Yeah we could do either one, the downside to having two deployments is we will need two clients, but we can create an abstraction over the clients
